modules/docs/src/03-protocols/02-simple-rest-json/03-client.md

```diff
@@ -42,3 +42,109 @@ object Clients {
+
+All smithy4s services provide an `X-Error-Type` in responses by default.
+
+#### Status Code
+
+If the `X-Error-Type` header is not defined, smithy4s clients will use the status code to attempt to decide which error type to utilize. It does so as follows:
+
+1. If there is a single Error type that contains the correct status code in the `httpError` trait, this type will be used. If there are two error types with the same status code, an `UnknownErrorResponse` will be surfaced to the client.
+2. If there is NOT a matching status code, but there is a single error marked with the `error` trait, this error type will be used as long as the returned status code is in the range for either a client or server error. In other words if a single error shape has no status code, but is annotated with `@error("client")` and the returned status code is 404 then this error type will be used. If there are multiple error types with no status code and a matching error type (client/server), then an `UnknownErrorResponse` will be surfaced to the client.
+
+Here are some examples to show more what this looks like.
+
+Example smithy model:
+
+```kotlin
+operation TestOp {
+  ...
+  errors: [NotFoundError, ServiceUnavailableError, CatchAllClientError, CatchAllServerError]
+}
+
+@httpError(404)
+@error("client")
+structure NotFoundError {
+  message: String
+}
+
+@httpError(503)
+@error("server")
+structure ServiceUnavailableError {
+  message: String
+}
+
+@error("client")
+structure CatchAllClientError {
+  message String
+}
+
+@error("server")
+structure CatchAllServerError {
+  message String
+}
+```
+
+And here are some scenarios using this example model. For all of these, assume that NO `X-Error-Type` header is provided.
+
+| Status Code | Error Selected          |
+|-------------|-------------------------|
+| 404         | NotFoundError           |
+| 400         | CatchAllClientError     |
+| 503         | ServiceUnavailableError |
+| 500         | CatchAllServerError     |
+
+However, if I added another error to the operation that looked like:
+
+```kotlin
+@error("client")
+structure AnotherError {
+  message: String
+}
+```
+
+then I would get the following:
+
+| Status Code | Error Selected           |
+|-------------|--------------------------|
+| 404         | NotFoundError            |
+| 400         | **UnknownErrorResponse** |
+| 503         | ServiceUnavailableError  |
+| 500         | CatchAllServerError      |
+
+Notice that the 400 status code cannot be properly mapped. This is because there is no exact match AND there are two errors that are labeled with `@error("client")` which also do not have an associated `httpError` trait containing a status code.
+
+If I add another error type to the operation that looks like:
+
+```kotlin
+@httpError(404)
+@error("client")
+structure AnotherNotFoundError {
+  message: String
+}
+```
+
+Then I will get the following:
```

Comment:
minor: most of the documentation is impersonal

by that I mean that we don't have many instances of `I do this` or `I get this`. I think we should keep the same style.

Otherwise good job on the explanations, it's well presented

Reply:
Good catch, I slipped into first person there. I will fix!